### PR TITLE
Additions to VRFv2plus setup-env script

### PR DESCRIPTION
**Pull request by ibrajer**
* added flag for specifying number of BHS keys
* added flag for specifying number of BHF keys
* removed obsolete flag in register key superscript
* added superscript for fetching commitment from the coordinator contract (request fulfillment status)
* minor helper function renaming

### CODE REVIEW

File: core/scripts/common/vrf/setup-envs/README.md [hunk 1]
```diff
@@ -75,6 +75,8 @@ go run . \
 --bhf-creds-file <path_to_file_with_creds> \
 --num-eth-keys=1 \
 --num-vrf-keys=1 \
+--num-bhs-sending-keys= 1 \
```
Comment (iljapavlovs) [inline core/scripts/common/vrf/setup-envs/README.md:78]: can you add this also to V2 example?

Comment (ibrajer) [reply core/scripts/common/vrf/setup-envs/README.md:78]: yes

File: core/scripts/common/vrf/setup-envs/README.md [hunk 2]
```diff
@@ -75,6 +75,8 @@ go run . \
 --bhf-creds-file <path_to_file_with_creds> \
 --num-eth-keys=1 \
 --num-vrf-keys=1 \
```
[the remark this answers is not in the document]
Comment (iljapavlovs) [reply core/scripts/common/vrf/setup-envs/README.md:77]: im ok with that, just I thought `VRF key` notion  is more widely used than `proving key`. CL node has a notion of VRF key

[the remark this answers is not in the document]
Comment (ibrajer) [reply core/scripts/common/vrf/setup-envs/README.md:79]: I left it the way it is in the end to avoid confusion...

**Review by iljapavlovs — APPROVED** [review]

**Pull request merged**